msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/ManagedIdentityParameters.java

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.aad.msal4j;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.experimental.Accessors;
+
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Object containing parameters for managed identity flow. Can be used as parameter to
+ * {@link ManagedIdentityApplication#acquireTokenForManagedIdentity(ManagedIdentityParameters)}
+ */
+@Builder
+@Accessors(fluent = true)
+@Getter
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+public class ManagedIdentityParameters implements IAcquireTokenParameters {
+
+    String resource;
+    
+    boolean forceRefresh;
+
+    IEnvironmentVariables environmentVariables;
+
+    public String getResource() {
```

Comment:
Do you need to add getters and setters if you used Lombok with builder / accessors etc. 

I would recommend that for public APIs you don't use Lombok magic, to at least be able to add comments?

Reply:
This would be a similar answer to https://github.com/AzureAD/microsoft-authentication-library-for-java/pull/661/files/d7364ba1a13185751f231ea9aaaef2deab3a8ede#r1311659704, but yeah getters aren't needed if you have Lombok's `@Getter` annotation, and the setters aren't needed if you have the `@Builder` annotation. These sorts of issues will be solved in the follow-up refactoring PR.